src/runtime/hexagon_remote/remote.cpp

```diff
@@ -21,6 +21,57 @@ extern "C" {
 typedef halide_hexagon_remote_handle_t handle_t;
 typedef halide_hexagon_remote_buffer buffer;
 
+#define MAX_WORKER_THREADS 4
+#define NUM_WORKER_THREADS_TO_CREATE 3
+#define STACK_SIZE 4096
```

Comment:
Currently, we put up to 16 kb allocations on the stack: https://github.com/halide/Halide/blob/master/src/CodeGen_Internal.cpp#L185

There might be several of them, and the stack gets used for other things too. So, maybe a few hundred kb at minimum?